cypress/e2e/news/createReadDeleteNewsTimeOptions.feature

```diff
@@ -0,0 +1,54 @@
+@stable_test
+@regression_test
+Feature:  News - To read a news on the respective dashboards
+
+    As a teacher I want to read the news shown on the dashboard so that I'm informed about the latest news
+
+    Scenario: User creates news with time options, reads them and deletes them
+        # as a pre-condition create users for author and reader
+        Given I am logged in as a '<news_reader>' at '<namespace>'
+        Given I am logged in as a '<news_author>' at '<namespace>'
+
+        # first user creates school news with different time options
+        When I go to news overview
+        And I click on add news button
```

Comment:
you should check 'And' is used I guess in all news features-

Reply:
This is copy and paste. I will change it here and also in the test I didn't edit.